themes/bootstrap3/templates/search/searchbox.phtml

```diff
@@ -1,3 +1,4 @@
+<a href="FOO" title="<?=$this->transEsc('"Phoo"')?>">Pho</a>
```

Comment:
I'm not sure what this is, but I'm guessing it's not supposed to be here. :-)

Reply:
Don't you like it? :D Err.. I'll remove it.

Reply:
Thanks! All tests are (unsurprisingly) passing, so I'll merge this soon.